### PR TITLE
Increase versions of OSRF packages

### PR DESCRIPTION
**Pull request by chapulina**
Updated style to make the file easier to read and pinned to specific versions - they match what's latest in packages.osrfoundation.org.

### CODE REVIEW

File: config/ignition_citadel_gazebo11_debian_buster.yaml [hunk 1]
```diff
@@ -3,4 +3,104 @@ method: http://packages.osrfoundation.org/gazebo/debian-stable
...
+(Package (% libignition-sensors3-thermal-camera, $Version (% 3.0.0*))|\
+(Package (% libignition-sensors3-thermal-camera-dev, $Version (% 3.0.0*))|\
+(Package (% libignition-tools-dev, $Version (% 1.0.0*))|\
+(Package (% libignition-transport8, $Version (% 8.0.0*))|\
+(Package (% libignition-transport8-core-dev, $Version (% 8.0.0*))|\
+(Package (% libignition-transport8-dbg, $Version (% 8.0.0*))|\
+(Package (% libignition-transport8-dev, $Version (% 8.0.0*))|\
+(Package (% libignition-transport8-log, $Version (% 8.0.0*))|\
+(Package (% libignition-transport8-log-dev, $Version (% 8.0.0*))|\
+(Package (% libogre-2.1)|\
```
Comment (j-rivero) [inline config/ignition_citadel_gazebo11_debian_buster.yaml:98]: Not sure if we want versions for ogre too, I can not think a reason not to do it:
```suggestion
(Package (% libogre-2.1, $Version (% 2.0.9999~20180616~06a386f-3osrf~*))|\
```

File: config/ignition_citadel_gazebo11_debian_buster.yaml [hunk 2]
```diff
@@ -3,4 +3,104 @@ method: http://packages.osrfoundation.org/gazebo/debian-stable
...
+(Package (% libogre-2.1-dev)|\
```
Comment (j-rivero) [inline config/ignition_citadel_gazebo11_debian_buster.yaml:99]:
```suggestion
(Package (% libogre-2.1, $Version (% 2.0.9999~20180616~06a386f-3osrf~*))|\
```

File: config/ignition_citadel_gazebo11_ubuntu_focal.yaml [hunk 3]
```diff
@@ -3,4 +3,104 @@ method: http://packages.osrfoundation.org/gazebo/ubuntu-stable
...
+(Package (% libignition-sensors3-thermal-camera, $Version (% 3.0.0*))|\
+(Package (% libignition-sensors3-thermal-camera-dev, $Version (% 3.0.0*))|\
+(Package (% libignition-tools-dev, $Version (% 1.0.0*))|\
+(Package (% libignition-transport8, $Version (% 8.1.0*))|\
+(Package (% libignition-transport8-core-dev, $Version (% 8.1.0*))|\
+(Package (% libignition-transport8-dbg, $Version (% 8.1.0*))|\
+(Package (% libignition-transport8-dev, $Version (% 8.1.0*))|\
+(Package (% libignition-transport8-log, $Version (% 8.1.0*))|\
+(Package (% libignition-transport8-log-dev, $Version (% 8.1.0*))|\
+(Package (% libogre-2.1)|\
```
Comment (j-rivero) [inline config/ignition_citadel_gazebo11_ubuntu_focal.yaml:98]:
```suggestion
(Package (% libogre-2.1, $Version (% 2.0.9999~20180616~06a386f-3osrf~*))|\
```

File: config/ignition_citadel_gazebo11_ubuntu_focal.yaml [hunk 4]
```diff
@@ -3,4 +3,104 @@ method: http://packages.osrfoundation.org/gazebo/ubuntu-stable
...
+(Package (% libogre-2.1-dev)|\
```
Comment (j-rivero) [inline config/ignition_citadel_gazebo11_ubuntu_focal.yaml:99]:
```suggestion
(Package (% libogre-2.1, $Version (% 2.0.9999~20180616~06a386f-3osrf~*))|\
```

**Review by j-rivero — APPROVED** [review]
Looks good to me, a couple of suggestions for versions in ogre.

**Comment by chapulina** [comment]
Thanks, @j-rivero, I didn't know how to handle the Ogre versions.

**Comment by nuclearsandwich** [comment]
@chapulina I pushed two changes, one fixing a missing paren that I think is also in the colcon config (and doesn't break anything due to the amount of sed involved in converting these into an aptly-consumed format) and removing the `|` from the final package line since that's an "OR" character in the query language. As long as nothing else looks wrong to you I think this is ready to go.

**Comment by nuclearsandwich** [comment]
Import jobs for ROS https://build.ros.org/job/import_upstream/148/ and ROS 2 https://build.ros2.org/job/import_upstream/304/

File: config/ignition_citadel_gazebo11_debian_buster.yaml [hunk 5]
```diff
@@ -3,4 +3,104 @@ method: http://packages.osrfoundation.org/gazebo/debian-stable
...
+(Package (% libignition-sensors3-thermal-camera-dev), $Version (% 3.0.0*))|\
+(Package (% libignition-tools-dev), $Version (% 1.0.0*))|\
+(Package (% libignition-transport8), $Version (% 8.0.0*))|\
+(Package (% libignition-transport8-core-dev), $Version (% 8.0.0*))|\
+(Package (% libignition-transport8-dbg), $Version (% 8.0.0*))|\
+(Package (% libignition-transport8-dev), $Version (% 8.0.0*))|\
+(Package (% libignition-transport8-log), $Version (% 8.0.0*))|\
+(Package (% libignition-transport8-log-dev), $Version (% 8.0.0*))|\
+(Package (% libogre-2.1), $Version (% 2.0.9999~20180616~06a386f-3osrf~*))|\
+(Package (% libogre-2.1), $Version (% 2.0.9999~20180616~06a386f-3osrf~*))|\
```
Comment (matthew-reynolds) [inline config/ignition_citadel_gazebo11_debian_buster.yaml:99]: @j-rivero @chapulina Shouldn't this second `libogre-2.1` be a `libogre-2.1-dev`? Looks like the `-dev` got dropped when adding the `$Version`.

(Same in the other yaml)

Comment (chapulina) [reply config/ignition_citadel_gazebo11_debian_buster.yaml:99]: Nice catch, see https://github.com/ros-infrastructure/reprepro-updater/pull/93

**Pull request merged**